src/mainboard/protectli/vault_kbl/devicetree.cb

```diff
@@ -184,17 +184,17 @@ chip soc/intel/skylake
 	register "PcieRpClkSrcNumber[8]" = "5"
 
 
-	# USB 2.0 enable ports 1-8, disable ports 9-12
+	# USB 2.0 enable ports 1-8 and 10, disable ports 9, 11-12
 	register "usb2_ports[0]"  = "USB2_PORT_SHORT(OC_SKIP)"	# TYPE-A Port
 	register "usb2_ports[1]"  = "USB2_PORT_SHORT(OC_SKIP)"	# TYPE-A Port
 	register "usb2_ports[2]"  = "USB2_PORT_SHORT(OC_SKIP)"	# TYPE-A Port
 	register "usb2_ports[3]"  = "USB2_PORT_SHORT(OC_SKIP)"	# TYPE-A Port
 	register "usb2_ports[4]"  = "USB2_PORT_SHORT(OC_SKIP)"	# Type-A Port
 	register "usb2_ports[5]"  = "USB2_PORT_SHORT(OC_SKIP)"	# TYPE-A Port
 	register "usb2_ports[6]"  = "USB2_PORT_SHORT(OC_SKIP)"	# TYPE-A Port
-	register "usb2_ports[7]"  = "USB2_PORT_SHORT(OC_SKIP)"  # FW6: mPCIe, FW6D: mSATA slot
+	register "usb2_ports[7]"  = "USB2_PORT_SHORT(OC_SKIP)"  # FW6: mPCIe, FW6D: mSATA1 slot
 	register "usb2_ports[8]"  = "USB2_PORT_EMPTY"		# Disabled
-	register "usb2_ports[9]"  = "USB2_PORT_SHORT(OC_SKIP)"  # mPCIe slot
+	register "usb2_ports[9]"  = "USB2_PORT_SHORT(OC_SKIP)"  # mPCIe slot, WIFI1 slot
```

Comment:
no need to add the number in the port marking, there is only 1 mPCIe WiFi port either way

Reply:
Removed the number from the port marking